acceptance-tests/src/test/scala/io/renku/graph/acceptancetests/data/Project.scala

```diff
@@ -27,18 +27,17 @@ import eu.timepit.refined.numeric.Positive
 import io.renku.graph.acceptancetests.data.Project._
 import io.renku.graph.model.entities.Project.ProjectMember
 import io.renku.graph.model.projects.{GitLabId, Name, Path}
-import io.renku.graph.model.testentities
+import io.renku.graph.model.{projects, testentities}
 import io.renku.tinytypes._
 import io.renku.tinytypes.constraints._
 
 import java.net.{MalformedURLException, URL}
-import java.time.Instant
 
 final case class Project(entitiesProject: testentities.RenkuProject,
                          id:              GitLabId,
                          maybeCreator:    Option[ProjectMember],
                          members:         NonEmptyList[ProjectMember],
-                         updatedAt:       DateUpdated,
+                         dateModified:    projects.DateModified,
```

Comment:
just wondering: the descriptions mentions to return `dateCreated` but here it's `dateModified`?

Reply:
Yes, my fault. Description should be saying `dateModified`.